packages/commons/src/config/EnvironmentVariablesService.ts

```diff
@@ -72,6 +86,13 @@ class EnvironmentVariablesService extends ConfigService {
     return truthyValues.includes(value.toLowerCase());
   }
 
+  private getXrayTraceData(): string[] {
```

Comment:
```suggestion
private getXrayTraceData(): { root: string, sampled: string } | undefined {
```

Reply:
The `string[]` return type is the signature you prescribed in the original issue.

Reply:
True, my bad.

After looking at the full implementation I think parsing and looking for the values would make for a more solid implementation rather than relying on the order of the fields. This way if fields get added or the order changes for some reason it won't immediately break.

What do you think?